backend/store/error_groups.go

```diff
@@ -48,15 +48,18 @@ func (store *Store) ListErrorObjects(errorGroup model.ErrorGroup, params ListErr
 	)
 
 	if params.After != nil {
-		query = query.Order("id DESC").Where("id < ?", *params.After)
+		query = query.Order("error_objects.id DESC").Where("error_objects.id < ?", *params.After)
```

Comment:
Ensures the column is not ambiguous when we search against `sessions.email` above.